examples/js/exporters/GLTFExporter.js

```diff
@@ -947,6 +953,21 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
+			// pbrSpecularGlossiness diffuse, specular and glossiness factor
+			if ( material.isGLTFSpecularGlossinessMaterial ) {
+				
+				var diffuseFactor = [ 1, 1, 1, 1 ];
+				material.color.toArray( diffuseFactor, 0 );
+				gltfMaterial.extensions.KHR_materials_pbrSpecularGlossiness.diffuseFactor = diffuseFactor;
```

Comment:
These three lines can be replaced with:

```js
if ( gltfMaterial.pbrMetallicRoughness.baseColorFactor ) {

  gltfMaterial.extensions.KHR_materials_pbrSpecularGlossiness.diffuseFactor = gltfMaterial.pbrMetallicRoughness.baseColorFactor;

}
```

That gets you opacity too, which is missed here otherwise.